airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2603,6 +2603,9 @@ components:
           format: uri
         icon:
           type: string
+        protocolVersion:
```

Comment:
Not sure what we have done in the past, but this has been named `protocol_version` in the protocol.  Should we match the case with the field in the protocol or have we been using camel case traditionally here in the API definition even if the protocol uses lower underscore?

Reply:
I am sticking camel case for consistency.
Our casing is a bit inconsistent at the moment, but a lot of it is in the protocol which would make fixing it a breaking change. We should plan that clean up at some point.